samples/bluetooth/peripheral_fast_pair/src/main.c

```diff
@@ -40,6 +41,21 @@ static struct k_work bt_adv_restart;
 static struct k_work_delayable fp_adv_mode_status_led_handle;
 static struct k_work_delayable fp_discoverable_adv_timeout;
 
+
+static void advertising_stop(void)
+{
+	int ret = k_work_cancel_delayable(&fp_discoverable_adv_timeout);
+
+	__ASSERT_NO_MSG((ret & ~(K_WORK_CANCELING)) == 0);
+	ARG_UNUSED(ret);
+
+	int err = bt_adv_helper_adv_stop();
+
+	if (err) {
+		LOG_ERR("Failed to stop advertising (err %d)", err);
+	}
```

Comment:
Can we reuse the error variable?

```suggestion
	ret = bt_adv_helper_adv_stop();
	if (ret) {
		LOG_ERR("Failed to stop advertising (err %d)", ret);
	}
```

Reply:
Changed. Initially I wanted to keep it separated, because `k_work_cancel_delayable` returns `k_work_delayable_busy_get` which is actually not an error value